packages/crud/src/vaadin-crud.js

```diff
@@ -247,14 +248,15 @@ class Crud extends ControllerMixin(ElementMixin(ThemableMixin(PolymerElement)))
         [part='scroller'] {
           display: flex;
           flex-direction: column;
-          overflow: auto;
           flex: auto;
         }
 
         [part='footer'] {
           display: flex;
           flex: none;
           flex-direction: row-reverse;
+          position: sticky;
+          bottom: 0;
```

Comment:
On macOS / iOS, if rubber-band scrolling is enabled, the toolbar can now be moved upwards due to overscroll. Should we change `overscroll-behavior` on the editor part so that you can not overscroll?

Reply:
Thanks, added `overscroll-behavior: none`. Couldn't test it locally since I'm still on Monterey and my XCode versuin has Simulator using iOS 15 that doesn't support this CSS property yet.

Reply:
Seems to work now, tested with Chrome, Safari, Firefox on macOS. On an iPhone it shouldn't be an issue since it should use the overlay. On an iPad it works with iOS 16 - is that the version we support with v24?

Reply:
In Vaadin 24 we still support Safari 15. However we already use `overscroll-behavior` in [app-layout](https://github.com/vaadin/web-components/blob/f38b8b3147510b80857d2b49ca30038f1033a059/packages/app-layout/src/vaadin-app-layout.js#L271).